### PR TITLE
Create Statistics View

### PR DESCRIPTION
**Pull request by wrac4242**
Implements #149 

currently draft PR

### CODE REVIEW

**Comment by wrac4242** [comment]
Current state of the PR, needs to have two changes done to fix the layout and threading to update

**Review by Founntain — APPROVED** [review]
As discussed in the Discord, I fixed the layouting.
For now I'm fine that the Date isn't live updating, we can implement this later in a seperate PR.

After doing this. I can say, that I'm ok with these changes and I have to say, for your first PR, you finished this task better than I excepted.

+ Not much code syntax erros
+ Code was clean
+ And was done as intented without any querky workarounds

Added @Cesan as reviewer so she can look with her eyes on this as well.

**Comment by wrac4242** [comment]
Changes LGTM, personally a few of the commits would be separate, but its fine

**Review by Cesan — APPROVED** [review]
PR looks good overall.
I changed some small stuff not necessarily needed which improves some performance and code quality.

File: OsuPlayer/Views/StatisticsViewModel.cs [hunk 1]
```diff
@@ -82,10 +82,15 @@ public StatisticsViewModel()
     private async void Block(CompositeDisposable disposable)
     {
         // Done separately so they can fail independently
-        await UpdateApiStatistics();
-        await UpdateBeatmapCount();
-        await UpdateStorageAmount();
-        
+        var tasks = new[]
```
Comment (Founntain) [inline OsuPlayer/Views/StatisticsViewModel.cs:85]: I like that

Comment (wrac4242) [reply OsuPlayer/Views/StatisticsViewModel.cs:85]: same

**Comment by wrac4242** [comment]
do merge when ready :)

**Comment by Cesan** [comment]
Merging. The basic view works fine if we need further features like live updates or stuff we'll do it in another PR.

**Pull request merged**